rust-runtime/aws-smithy-eventstream/src/frame.rs

```diff
@@ -22,6 +25,17 @@ const MESSAGE_CRC_LENGTH_BYTES: u32 = size_of::<u32>() as u32;
 const MAX_HEADER_NAME_LEN: usize = 255;
 const MIN_HEADER_LEN: usize = 2;
 
+pub(crate) const TYPE_TRUE: u8 = 0;
+pub(crate) const TYPE_FALSE: u8 = 1;
+pub(crate) const TYPE_BYTE: u8 = 2;
+pub(crate) const TYPE_INT16: u8 = 3;
+pub(crate) const TYPE_INT32: u8 = 4;
+pub(crate) const TYPE_INT64: u8 = 5;
+pub(crate) const TYPE_BYTE_ARRAY: u8 = 6;
+pub(crate) const TYPE_STRING: u8 = 7;
+pub(crate) const TYPE_TIMESTAMP: u8 = 8;
+pub(crate) const TYPE_UUID: u8 = 9;
+
```

Comment:
Just a general question, so we do not allow a float / double as a type in HeaderValue? I guess, they can always be passed as a String.